.github/workflows/wiki.yml

```diff
@@ -0,0 +1,18 @@
+name: Publish Wiki
+
+on:
+  push:
+    branches:
+      - development
+    paths:
+      - .github/wiki/**
+
+jobs:
+  publish:
+    name: Publish wiki
+    uses: nimblehq/github-actions-workflows/.github/workflows/publish_wiki.yml@0.1.0
```

Comment:
Can we not do `nimblehq/publish-github-wiki-action@v1.0` according to the [readme](https://github.com/nimblehq/publish-github-wiki-action#usage) now?